BigIntegerTests/Methods.cs

```diff
@@ -79,5 +79,64 @@ public void TestGetBytes()
                 Assert.AreEqual(zero_bytes[0], (byte)0);
             }
         }
+
+
+        [TestMethod()]
+        public void modPowTest()
```

Comment:
Please, maintain the naming style used in the project. That should be `TestModPow`.